test/lib/next-test-utils.ts

```diff
@@ -1500,3 +1511,15 @@ export async function getStackFramesContent(browser) {
 
   return stackFramesContent
 }
+
+export async function toggleCollapseCallStackFrames(browser: BrowserInterface) {
+  const button = await browser.elementByCss('[data-expand-ignore-button]')
+  const lastExpanded = await button.getAttribute('data-expand-ignore-button')
+  await button.click()
+
+  await retry(async () => {
+    const button = await browser.elementByCss('[data-expand-ignore-button]')
```

Comment:
Doesn't this already have built-in retry? If it does, move it out of the retry block.